src/TreeSitter/Node.hs

```diff
@@ -55,8 +52,8 @@ pokeStruct a = Struct (\ p -> do
 
 
 instance Storable Node where
-  alignment _ = alignment (TSNode nullPtr 0 0 :: TSNode)
-  sizeOf _ = 56
+  alignment _ = alignment (TSNode 0 0 0 0 nullPtr nullPtr :: TSNode)
+  sizeOf _ = 72
```

Comment:
This accounts for the changes to the size of the `TSNode` inside `Node`, correct?

Reply:
@robrix yep, the size of `TSNode` is 16 bytes larger now.